scss/_forms.scss

```diff
@@ -567,7 +567,7 @@ button.close {
         justify-content: center;
         color: $editor-background;
         &:hover, &.in-use{
-            filter: drop-shadow(3px 3px 1px $dark-pink);
+            filter: drop-shadow(2px 2px 1px $dark-pink);
```

Comment:
yeah, no, i went back and forth on this too

Reply:
I almost left it at 2.5

Reply:
it's v v subtle

Reply:
without actually knowing if that registers with the browser as different than 3

Reply:
_Shadow Nerds_